app/uk/gov/hmrc/servicedependencies/controller/ServiceDependenciesController.scala

```diff
@@ -206,7 +217,7 @@ class ServiceDependenciesController @Inject()(
       ).merge
     }
 
-  private def getRepositoryInfo(meta: MetaArtefact, latestVersions: Seq[LatestVersion], bobbyRules: BobbyRules) = {
+  private def toRepositoryInfo(meta: MetaArtefact, latestVersions: Seq[LatestVersion], bobbyRules: BobbyRules) = {
```

Comment:
can we call it `toRepository` to match the type it returns?

Reply:
Done